packaging/src/main/assembly/src.xml

```diff
@@ -41,6 +41,12 @@
         <exclude>**/.project</exclude>
         <exclude>**/.settings/**</exclude>
         <exclude>**/thirdparty/**</exclude>
+        <exclude>standalone-metastore/metastore-common/src/gen/version/**</exclude>
+        <exclude>standalone-metastore/metastore-server/src/gen/**</exclude>
+        <exclude>common/src/gen/**</exclude>
+        <exclude>kafka-handler/src/test/gen/**</exclude>
+        <exclude>conf/hive-default.xml.template</exclude>
```

Comment:
Can the `hive-default.xml.template` be generalized or not?

Reply:
I do not think it is worth to generalize this.
The file is generated by https://github.com/apache/hive/blob/master/common/src/java/org/apache/hadoop/hive/ant/GenHiveTemplate.java

And the name is defined here:
https://github.com/apache/hive/blob/2524c2137bc2e70bd32e43a49a15fbfc0fa93159/common/pom.xml#L310-L314

So I think this is a one off file.